thirdparty/cmdconfig/commands/cmdsource/cmdsource.go

```diff
@@ -58,6 +63,9 @@ type SourceRunner struct {
 }
 
 func (r *SourceRunner) runE(c *cobra.Command, args []string) error {
+	if r.Output != cmdutil.Stdout && r.Output != cmdutil.Unwrap {
+		return fmt.Errorf("invalid input for --output flag, allowed values: %s|%s", cmdutil.Stdout, cmdutil.Unwrap)
```

Comment:
Pl. follow error style guide `docs/style-guides/errors.md` for error text.

Reply:
Updated.